package.json

```diff
@@ -34,13 +34,17 @@
   "author": "fe-app",
   "license": "MIT",
   "dependencies": {
-    "@lion/button": "^0.8.11",
+
+    "@lion/button": "^0.8.10",
     "@lion/form": "^0.7.18",
+    "@lion/form-core": "^0.8.4",
     "@lion/input": "^0.11.1",
     "@lion/localize": "^0.16.0",
+    "@lion/select": "^0.9.4",
     "@vaadin/router": "^1.7.4",
     "lit-element": "^2.0.1",
-    "lit-html": "^1.0.0"
+    "lit-html": "^1.0.0",
+    "node-modules": "^1.0.1"
```

Comment:
this one not needed, remove it, i already told to @alekhya66 why we dont need this one

Reply:
will remove it